docs/docs/resources/pipeline-components/kafka-source-connector.yaml

```diff
@@ -14,31 +14,31 @@
       ${pipeline_name}-output-topic:
         type: output # required
         # role: topic-role # only used if type is `extra`
-        keySchema: key-schema # must implement SchemaProvider to use
-        valueSchema: value-schema
+        key_schema: key-schema # must implement SchemaProvider to use
+        value_schema: value-schema
         partitions_count: 1
         replication_factor: 1
         configs: # https://kafka.apache.org/documentation/#topicconfigs
           cleanup.policy: compact
     models: # SchemaProvider is initiated with the values given here
       model: model
-  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # Pipeline prefix that will prefix every component name. If you wish to not
   # have any prefix you can specify an empty string.
   prefix: ${pipeline_name}-
   # Helm repository configuration
-  repoConfig:
-    repositoryName: my-repo # required
+  repo_config:
+    repository_name: my-repo # required
     url: https://bakdata.github.io/kafka-connect-resetter/ # required
-    repoAuthFlags: 
+    repo_auth_flags:
       username: user
       password: pass
       ca_file: /home/user/path/to/ca-file
       insecure_skip_tls_verify: false
   version: "1.0.4" # Helm chart version
   # Overriding Kafka Connect Resetter Helm values. E.g. to override the
   # Image Tag etc.
-  resetterValues:
+  resetter_values:
     imageTag: "1.2.3"
   # offset.storage.topic
   # https://kafka.apache.org/documentation/#connect_running
-  offsetTopic: offset_topic
+  offset_topic: offset_topic
```

Comment:
If this is part of the Helm values, please keep the camelCase format.
https://github.com/bakdata/kafka-connect-resetter/blob/fc838e560d6b29302ed1e04c0b59748ea109d0f2/charts/kafka-connect-resetter/values.yaml#L23

Reply:
hm, I wonder why this is not part of the `resetter_values` then?

Reply:
at least it's not part of the Helm values in this context

Reply:
Shouldn't this be formatted like this:
```yaml
  resetter_values:
    imageTag: "1.2.3"
    config:
        offsetTopic: offset_topic
```
I found this `offset_topic`: https://github.com/bakdata/kpops/blob/85d4d048946753dabb2e5aa17052aafcb6d9b7e2/kpops/components/base_components/kafka_connector.py#L322
I am not quite sure if this is needed 🤔 
Becuase we have the one that represents the helm value here: https://github.com/bakdata/kpops/blob/85d4d048946753dabb2e5aa17052aafcb6d9b7e2/kpops/component_handlers/kafka_connect/model.py#L63

Reply:
this is assigned automatically by KPOps though: https://github.com/bakdata/kpops/blob/599980add1eb8237b36b0a5b7102075ed7ebbe09/kpops/components/base_components/kafka_connector.py#L366

Reply:
So now I can write something like this. Correct? 
```yaml
  resetter_values:
    imageTag: "1.2.3"
    config:
        offsetTopic: offset_topic
  offset_topic: offset_topic <--------- this seems redundant
```
Wouldn't it make sense to have only one definition?

Reply:
first and foremost the offset topic is a Source connector config actually, see `offset.storage.topic` https://kafka.apache.org/documentation/#connectconfigs_offset.storage.topic
as such, it could be written as

```yaml
- type: kafka-source-connector
  name: example
  app:
    "offset.storage.topic": offsets
```

I guess we made it a top-level attribute to make it more clear that this is required for source connectors only. If we decide to change something about it, I think it should be a separate issue/PR